From 5fe6a52af1062ecdc655158eb3e6894f3da9fd0d Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 20 Jun 2024 11:09:31 -0700
Subject: [PATCH 09/10] Disable protonmsg plugin tests

Not used in Clear Linux, so don't run the test.

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 requirements.txt                                                | 2 --
 .../{test_protonmsg.py => test_protonmsg.py.do-not-test}        | 0
 2 files changed, 2 deletions(-)
 rename tests/test_plugins/{test_protonmsg.py => test_protonmsg.py.do-not-test} (100%)

diff --git a/requirements.txt b/requirements.txt
index a9f6c8e..6f2f506 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -3,6 +3,4 @@ Cheetah;python_version < '3.0'
 Cheetah3;python_version >= '3.0'
 psycopg2;python_version >= '3.0'
 python-multilib
-python-qpid-proton<0.38.0;python_version < '3.0'
-python-qpid-proton;python_version >= '3.0'
 defusedxml
diff --git a/tests/test_plugins/test_protonmsg.py b/tests/test_plugins/test_protonmsg.py.do-not-test
similarity index 100%
rename from tests/test_plugins/test_protonmsg.py
rename to tests/test_plugins/test_protonmsg.py.do-not-test
-- 
2.45.1

